### PR TITLE
docs(translation): missing fr pages

### LINKED ISSUE #1758: French translations
Missing translations:

- [x] https://nuxtjs.org/fr/integrations/deployments/layer0
- [x] https://nuxtjs.org/fr/integrations/deployments/cloudflare
- [x] https://nuxtjs.org/fr/integrations/deployments/dokku
- [x] https://nuxtjs.org/fr/integrations/deployments/fume
- [ ] https://nuxtjs.org/fr/integrations/deployments/github-pages
- [ ] https://nuxtjs.org/fr/integrations/deployments/google-appengine
- [ ] https://nuxtjs.org/fr/integrations/deployments/google-cloud-run
- [x] https://nuxtjs.org/fr/integrations/deployments/heroku
- [ ] https://nuxtjs.org/fr/integrations/deployments/hostman
- [ ] https://nuxtjs.org/fr/integrations/deployments/koyeb
- [ ] https://nuxtjs.org/fr/integrations/deployments/nginx
- [ ] https://nuxtjs.org/fr/integrations/deployments/platformsh
- [x] https://nuxtjs.org/fr/integrations/deployments/pm2
- [x] https://nuxtjs.org/fr/integrations/deployments/qovery
- [x] https://nuxtjs.org/fr/integrations/deployments/stormikitio
- [x] https://nuxtjs.org/fr/case-studies
- [ ] Docs Aside (title)
- [x] https://nuxtjs.org/fr/partners
- [x] https://nuxtjs.org/fr/docs/configuration-glossary/configuration-components

### PR DESCRIPTION
**Pull request by Geminii**
Resolves #1758 

Add translation for : 

- [x] https://nuxtjs.org/fr/integrations/deployments/layer0
- [x] https://nuxtjs.org/fr/integrations/deployments/cloudflare
- [x] https://nuxtjs.org/fr/integrations/deployments/dokku
- [x] https://nuxtjs.org/fr/integrations/deployments/fume
- [x] https://nuxtjs.org/fr/integrations/deployments/qovery
- [x] https://nuxtjs.org/fr/integrations/deployments/heroku
- [x] https://nuxtjs.org/fr/integrations/deployments/stormikitio
- [x] https://nuxtjs.org/fr/case-studies
- [x] https://nuxtjs.org/fr/partners
- [x] https://nuxtjs.org/fr/docs/configuration-glossary/configuration-components

### CODE REVIEW

File: content/fr/case-studies/0.github.md [hunk 1]
```diff
@@ -1,55 +1,55 @@
...
+  Le [load balancer GitHub](https://github.blog/2018-08-08-glb-director-open-source-load-balancer/) que nous avons open sourcé en 2018. C'est notre solution d'équilibreur de charge évolutive pour les centres de données bare metal et il alimente la majorité du trafic web et git de GitHub ainsi qu'il fait face à certains des principaux systèmes Intel. 
 
-  [GitHub Docs](https://github.blog/2020-10-14-how-we-open-sourced-docs-github-com/) which we open sourced in mid October 2020. With this effort, we're able to source new ideas from a broader and diverse set of individuals, especially from those who are  experts in the community and GitHub.
+  [GitHub Docs](https://github.blog/2020-10-14-how-we-open-sourced-docs-github-com/) que nous avons open sourcé en mi-octobre 2020. Avec cet effort, nous sommes capable de trouver de nouvelles idées auprès d'un ensemble plus large et diversifié d'individus, en particulier de ceux qui sont des experts de la communauté et de GitHub.
 
-## What is GitHub Stars?
+## Quest-ce que GitHub Stars?
```
Comment (roiLeo) [inline content/fr/case-studies/0.github.md:23]: missing white space after "Start"

File: content/fr/case-studies/0.github.md [hunk 2]
```diff
@@ -1,55 +1,55 @@
...
+Nuxt était le candidat idéal en raison du fait qu'il peut bien fonctionner en tant que site Web entièrement statique, mais vous pouvez progressivement le transformer en un site Web entièrement dynamique. Nous avons pensé qu'à l'avenir, nous ne serions peut-être pas satisfaits d'avoir un peu de retard en raison de devoir régénérer le site lorsque des modifications sont apportées dans le tableau de bord d'administration. Nous aimions déjà Vue et Nuxt mais cette fonctionnalité en particulier nous a été très utile. Nous avions l'assurance qu'à tout moment nous pouvions changer d'approche.
 
-## Are you using dynamic or static rendering? Why?
+## Utilisez-vous le rendu dynamique ou statique ? Pourquoi?
```
Comment (roiLeo) [inline content/fr/case-studies/0.github.md:39]: missing white space before last "?"

File: content/fr/case-studies/2.stores.md [hunk 3]
```diff
@@ -1,46 +1,47 @@
 ---
 template: post
 title: Stores.jp
-description: "stores.jp is an e-commerce platform which allows users to create their own e-commerce website. It focuses on users who don't have their own e-commerce site yet. That's why most of the features are designed so users can change the style and functionality easily without having any technical background knowledge."
+description: "stores.jp est une plateforme de e-commerce qui permet aux utilisateurs de créer leur propre site Web de e-commerce. Il se concentre sur les utilisateurs qui n'ont pas encore leur propre site de e-commerce. C'est pourquoi la plupart des fonctionnalités sont conçues pour que les utilisateurs puissent facilement modifier le style et les fonctionnalités sans avoir aucune connaissance technique."
```
Comment (roiLeo) [inline content/fr/case-studies/2.stores.md:4]: I would change "de e-commerce" to "d'e-commerce"

File: content/fr/case-studies/2.stores.md [hunk 4]
```diff
@@ -1,46 +1,47 @@
...
 imgUrl: img/case-studies/stores/cover.png
 headingImg:
   hidden: true
 ---
 
 ![Stores.jp homepage](img/case-studies/stores/main.png)
 
-## What is Stores.jp?
+## Qu'est-ce que Stores.jp?
```
Comment (roiLeo) [inline content/fr/case-studies/2.stores.md:12]: missing whitespace before "?"

File: content/fr/case-studies/2.stores.md [hunk 5]
```diff
@@ -1,46 +1,47 @@
...
 
-[stores.jp](http://stores.jp) is an e-commerce platform which allows users to create their own e-commerce website. It focuses on users who don't have their own e-commerce site yet. That's why most of the features are designed so users can change the style and functionality easily without having any technical background knowledge. 
+[stores.jp](http://stores.jp) est une plateforme de e-commerce qui permet aux utilisateurs de créer leur propre site Web de e-commerce. Il se concentre sur les utilisateurs qui n'ont pas encore leur propre site de e-commerce. C'est pourquoi la plupart des fonctionnalités sont conçues pour que les utilisateurs puissent facilement modifier le style et les fonctionnalités sans avoir aucune connaissance technique.
```
Comment (roiLeo) [inline content/fr/case-studies/2.stores.md:14]: same as said before I would change "de e-commerce"

File: content/fr/case-studies/3.parent-scheme.md [hunk 6]
```diff
@@ -1,60 +1,60 @@
...
+description: "Parent Scheme est une start-up basée au Royaume-Uni qui travaille dans le monde entier avec des organisations pour les aider à soutenir les parents qui travaillent. Il propose des livestreams, des ressources, des articles, des vidéos et du coaching via une plateforme interactive."
 imgUrl: img/case-studies/parent-scheme/cover.png
 headingImg:
   hidden: true
 ---
 
 ![Parent Scheme homepage](img/case-studies/parent-scheme/main.png)
 
-## What is Parent Scheme?
+## Qu'est-ce que Parent Scheme?
```
Comment (roiLeo) [inline content/fr/case-studies/3.parent-scheme.md:12]: missing whitespace before "?"

File: content/fr/case-studies/5.404-place-vendome.md [hunk 7]
```diff
@@ -1,44 +1,44 @@
...
   hidden: true
 ---
 
 ![404 Place Vendôme homepage](img/case-studies/404-place-vendome/main.png)
 
-## What is 404 Place Vendôme ?
+## Qu'est-ce que 404 Place Vendôme ?
 
-404 Place Vendôme is a jewellery company which was created in 2016 by two people. One of the founders father was a founder of gold and metals. I joined the team, a few months after at the beginning of 2016. They asked for me to make, a configurator of products for the jewellery design. And then I made the website using 3JS and 3D and vanilla JS. At the time I didn't knew about, react or Vue. I showed them the concept I made with my godfather, and they were really impressed.
+[404 Place Vendôme](https://404placevendome.com/) est une maison de joaillerie créée en 2016 par deux personnes. L'un des pères fondateurs était un fondateur d'or et de métaux. J'ai rejoint l'équipe, quelques mois après début 2016. Ils m'ont demandé de réaliser, un configurateur de produits pour la création de bijoux. Et puis j'ai fait le site web en utilisant threeJS et 3D et vanilla JS. À l'époque, je ne connaissais pas, React ou Vue. Je leur ai montré le concept que j'avais fait avec mon parrain, et ils ont été vraiment impressionnés.
```
Comment (roiLeo) [inline content/fr/case-studies/5.404-place-vendome.md:14]: I would change "À l'époque, je ne connaissais pas, React ou Vue" by "À l'époque, je ne connaissais ni react ni Vue."

File: content/fr/docs/5.configuration-glossary/6.configuration-components.md [hunk 8]
```diff
@@ -1,21 +1,266 @@
...
 
-- Type: `Boolean` or `Object`
+::alert
+Il est possible d'utiliser cette fonctionnalité avec Nuxt 2.10 - 2.12. Installez manuellement en ajoutant `@nuxt/components` dans `buildModules` à l'intérieur de `nuxt.config`.
+::
+
+- Type: `Boolean` ou `Object`
 - Default: `false`
 
 Lorsqu'il est défini à `true` ou qu'il utilise un objet, il inclura les dépendances du [composant](https://github.com/nuxt/components) et importera automatiquement vos composants (définis dans `~/components`) lorsque vous les utilisez dans vos modèles.
```
Comment (roiLeo) [inline content/fr/docs/5.configuration-glossary/6.configuration-components.md:21]: should we translate "component" to "composant" everywhere ?

File: content/fr/integrations/deployments/2.layer0.md [hunk 9]
```diff
@@ -42,9 +42,9 @@ npm i -g @layer0/cli
 
 </code-group>
 
-## Configure your project
+## Configurer votre projet
 
-3. Make sure [server side rendering is enabled](/docs/configuration-glossary/configuration-ssr) and in your `nuxt.config.js` add the `@layer0/nuxt` module:
+3. Assurez-vous que [le server side rendering est activé](/docs/configuration-glossary/configuration-ssr) et dans votre `nuxt.config.js` ajoutez le module `@layer0/nuxt` :
```
Comment (roiLeo) [inline content/fr/integrations/deployments/2.layer0.md:47]: Should we translate "server side rendering" to "rendu côté serveur" ?

File: content/fr/integrations/deployments/fume.md [hunk 10]
```diff
@@ -1,36 +1,38 @@
...
 
-## Setup
+- Structures à moindre coût prenant en charge à la fois Server et Static avec Lambda et CloudFront.
+- [Automated](https://github.com/marketplace/actions/fume-deployment) Déploiements avec rollbacks en un clic
+- Métriques et prévision des coûts pour chaque environnement
+- Contrôle de domaine - importer des hôtes, émettre des certificats et mapper l'enregistrement aux environnements
+- Notifications intégrées à Slack, Discord et autres plateformes de collaboration
 
-Get a production-ready URL in 2 minutes with these steps:
+## Installer
```
Comment (roiLeo) [inline content/fr/integrations/deployments/fume.md:29]: I would change with "Installation"

File: content/fr/integrations/deployments/qovery.md [hunk 11]
```diff
@@ -1,70 +1,70 @@
...
-Skip adding services for static website.
+Ignorez l'ajout de services pour le site Web statique.
 
-Click on "Deploy".
+Cliquez sur "Déployer".
 
-## Deploy
+## Déployer
 
 Your app should be deployed. You can see the status in real time by clicking on deployment logs.
```
Comment (roiLeo) [inline content/fr/integrations/deployments/qovery.md:58]: Could be translated: "Votre application devrait être déployée. Vous pouvez voir le statut en temps réel en cliquant sur les journaux de déploiement."

File: content/fr/integrations/deployments/stormkitio.md [hunk 12]
```diff
@@ -1,43 +1,41 @@
...
+Créez, déployez et faites évoluer facilement vos applications Nuxt avec [Stormkit.io](https://www.stormkit.io). Il prend en charge les roolbacks instantanées, la logique côté serveur, les injections d'extraits de code, plusieurs environnements de développement et plus encore...
 
-## Prerequisites
+## Conditions préalables
 
-This guide assumes you already have a Nuxt project to deploy. If you need a project, use the [create-nuxt-app](https://github.com/nuxt/create-nuxt-app) to get started or fork Stormkit's [Nuxt Example](https://github.com/stormkit-dev/hackernews-nuxt) before continuing.
+Ce guide suppose que vous avez déjà un projet Nuxt à déployer. Si vous avez besoin d'un projet, utilisez [create-nuxt-app](https://github.com/nuxt/create-nuxt-app) pour commencer ou forker [l'exemple Nuxt de Stormkit](https://github.com/stormkit-dev/hackernews-nuxt) avant de continuer.
 
-## Setup
+## Installer
```
Comment (roiLeo) [inline content/fr/integrations/deployments/stormkitio.md:23]: I would change with "Installation"

File: content/fr/partners/index.md [hunk 13]
```diff
@@ -1,8 +1,8 @@
 ---
 template: PageList
-title: Partners
-description: "Nuxt is an MIT licensed open source project and free to use. However, the maintenance effort is not sustainable without proper financial backing."
-heroTitle: Partners
-heroDescription: "Find your perfect match. Get help from a solutions partner or solutions provider."
+title: Partenaires
+description: "Nuxt est un projet open source sous licence MIT et gratuit à utiliser. Cependant, l'effort d'entretien n'est pas durable sans un soutien financier approprié."
```
Comment (roiLeo) [inline content/fr/partners/index.md:4]: I would change "gratuit à utiliser" with "libre d'utilisation"

**Review by roiLeo — CHANGES_REQUESTED** [review]

File: content/fr/docs/5.configuration-glossary/6.configuration-components.md (hunk 8 above)
Comment (thomasbnt) [reply content/fr/docs/5.configuration-glossary/6.configuration-components.md:21]: It seems more logical to put the same word everywhere

**Comment by thomasbnt** [comment]
Hmmm maybe I made a mistake with Git... I created a new PR https://github.com/nuxt/nuxtjs.org/pull/1819 but if it can be merged on this one so that everything is concentrated on one, I want help 😂

File: content/fr/case-studies/0.github.md (hunk 1 above)
Comment (Geminii) [reply content/fr/case-studies/0.github.md:23]: @roiLeo Thanks for this :) 
Feel free next time to `Insert suggestion`. More efficient to accept your purpose 😇

File: content/fr/docs/5.configuration-glossary/6.configuration-components.md (hunk 8 above)
Comment (Geminii) [reply content/fr/docs/5.configuration-glossary/6.configuration-components.md:21]: In a sentence, yes we need to translate it but not for references.

File: content/fr/integrations/deployments/qovery.md (hunk 11 above)
Comment (Geminii) [reply content/fr/integrations/deployments/qovery.md:58]: I omit this, thanks a lot 🙏

**Comment by Geminii** [comment]
> Hmmm maybe I made a mistake with Git... I created a new PR #1819 but if it can be merged on this one so that everything is concentrated on one, I want help 😂

@thomasbnt not a mistake and thanks for your contribution 🙏 
Can you try to change the destination of your MR to `docs/missing-french-translation` instead of `main` ?

**Review by roiLeo — APPROVED** [review]

**Pull request closed without merging**